### PR TITLE
new icon: datagrip (original, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR contributes to #1125**

## Link to prove your SVG is correct and up-to-date.

https://www.jetbrains.com/fr-fr/datagrip/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -1462,6 +1462,27 @@
         "color": "#A80030",
         "aliases": []
     },
+    {
+        "name": "datagrip",
+        "altnames": [],
+        "tags": [
+            "jetbrains",
+            "ide"
```
Comment (Snailedlt) [inline devicon.json:1470]:
```suggestion
            "ide",
            "sql",
            "database"
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Suggested some more tags

File: devicon.json [hunk 2]
```diff
@@ -1562,6 +1562,29 @@
         "color": "#A80030",
         "aliases": []
     },
+    {
```
Comment (Snailedlt) [inline devicon.json:1565]: The new icon needs to be in the correct alphabetical order compared to the other icons

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 

Perfection is not attainable, but if we chase perfection we can catch excellence.
\- [Vince Lombardi](https://www.brainyquote.com/authors/vince-lombardi-quotes)

**Comment by kilian-paquier** [comment]
@Snailedlt Here too

**Pull request merged**